components/AppHeader.vue

```diff
@@ -201,9 +217,27 @@ export default class AppHeader extends Vue {
         wallet: accounts[0].address,
         method,
       })
-      return this.initWallet(connection)
+      await this.initWallet(connection)
+      if (!this.currentAddress || !this.signer) return
+      const signature = await this.signMessageMemo('authorize', [
+        'profile',
+        'read:nftbook',
+        'write:nftbook',
+        'read:nftcollection',
+        'write:nftcollection',
```

Comment:
Maybe refactor to a constant for easier maintainance

Reply:
[🎨 Add constant for authorization permission](https://github.com/likecoin/app-like-co/pull/474/commits/5c2c13370611651bc56f227ddcdf07466b7ea02a)